src/main/java/dk/cngroup/lentils/controller/LoginController.java

```diff
@@ -11,4 +14,14 @@ public String login() {
         return "login/login";
     }
 
+    @GetMapping("/")
+    public String index() {
+        return "redirect:/login";
+    }
+
+    @GetMapping("/successfulLogin")
+    public String redirectUserBasedOnRole() {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        return LoginFactory.getRedirectUrlAfterLoginForRole(authentication.getAuthorities().toString());
```

Comment:
Asi bych to nenazyval "factory" a neposilal bych tam string. Navrhuju rename:
`LoginUtils.getRedirectUrlAfterLoginByRole(authentication.getAuthorities());`
To porovnavani v tom case v soucasne LoginFactory je destastne protoze to porovnava string pole. Ale Java je na toto blba a elengantne to nejde udelat na par radku, takze ti to odpustim, ale presun to `toString()` az do te metody v te factory (At je ten skaredy kod separovany od toho pekneho).

Reply:
Souhlas, tim padem bych mel vytvorit novou package Utility?
string oddelam :)